tests/common.cpp

```diff
@@ -316,6 +316,23 @@ result_t validateFloatEpsilon(__m128 a,
     float df1 = fabsf(t[1] - f1);
     float df2 = fabsf(t[2] - f2);
     float df3 = fabsf(t[3] - f3);
+
+    if ((std::isnan(t[0]) && std::isnan(f0)) || (t[0] == 0 && f0 == 0)) {
```

Comment:
Add some comments which describe the motivation.

Reply:
Done.